controller/api/models.py

```diff
@@ -381,7 +381,7 @@ def _healthcheck(self, containers, config):
         # HACK (bacongobbler): we need to wait until publisher has a chance to publish each
         # service to etcd, which can take up to 20 seconds.
         time.sleep(20)
-        for i in range(0, 4):
+        for i in range(len(intervals)):
```

Comment:
I think `len(intervals)` would get evaluated each iteration, even though the length doesn't change... so wouldn't it be better to evaluate this once and then use the result? *super* minor, though.